packages/MapFactory/src/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+#
+# A) Package-specific configuration options
+#
+
+TRIBITS_CONFIGURE_FILE(${PACKAGE_NAME}_config.h)
```

Comment:
File is empty.  Remove `DataTransferKitMapFactory_config.h.in` and comment this out.
I'd like to move away from every package having its own `config.h`.